### PR TITLE
Added gaia worker documentation

### PR DESCRIPTION
**Pull request by michelvocks**
Related PR: https://github.com/gaia-pipeline/gaia/pull/166

### CODE REVIEW

**Comment by Skarlso** [comment]
Oh cool! Been waiting fro this. :D I will meticulously read it. :P

**Comment by Skarlso** [comment]
Oh what's hard to notice here is that I fixed a few typos and reworded a few things here and there.

But the damn indentation messed up the difference. :/ Sorry about that man. I thought it would be better visible that way.

**Review by Skarlso — APPROVED** [review]

File: content/operations/worker/_index.md
Comment (michelvocks) [inline content/operations/worker/_index.md]:
```suggestion
Once a worker has been registered at your primary Gaia instance, new pipeline runs will be automatically scheduled to
```

File: content/operations/worker/_index.md
Comment (michelvocks) [inline content/operations/worker/_index.md]:
```suggestion
no further pipeline runs will be scheduled to this node till the node is active again. Additionally, the primary Gaia
```

File: content/rfcs/0001_distributed_execution_of_pipelines.md
Comment (Skarlso) [inline content/rfcs/0001_distributed_execution_of_pipelines.md]: There are some very weird indentations in this. :D Can you fix that ? After that it's cool. :)

**Review by Skarlso — APPROVED** [review]

**Comment by Skarlso** [comment]
👍 Good stuff man

**Comment by Skarlso** [comment]
@michelvocks This can be merged, right?

**Comment by michelvocks** [comment]
@Skarlso Yeah I wanted to merge that with the main PR https://github.com/gaia-pipeline/gaia/pull/166
Just waiting for your final approval 😄

**Pull request merged**
